libnetwork/Host.h

```diff
@@ -160,14 +199,17 @@ class Host : public std::enable_shared_from_this<Host>
     std::string m_listenHost = "";
     uint16_t m_listenPort = 0;
 
-    std::function<void(NetworkException, NodeID, std::shared_ptr<SessionFace>)> m_connectionHandler;
+    std::function<void(NetworkException, NodeInfo const&, std::shared_ptr<SessionFace>)>
+        m_connectionHandler;
 
     bool m_run = false;
 
     std::shared_ptr<std::thread> m_hostThread;
 
     // certificate rejected list of nodeID
     std::vector<std::string> m_crl;
+
+    static std::string split_char;

```

Comment:
why this "split_char" need to be static?

Reply:
To improve the maintainability of the code

Reply:
done, please refer to: https://github.com/FISCO-BCOS/FISCO-BCOS/pull/600/files#diff-df0545cce20878ff6751b98f9348901cR212